### PR TITLE
Final Project Begins

### PR DESCRIPTION
**Pull request by nihalerdal**
Here is my Final Project Begins pull request for the review.

### CODE REVIEW

File: src/Components/TodoListItem/TodoListItem.jsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import { FaTrash } from "react-icons/fa";
+import { FaEdit } from "react-icons/fa";
+import { FaSave } from "react-icons/fa";
+import PropTypes from "prop-types";
+import { useState } from "react";
+
+function TodoListItem({ title, onRemoveTodo, id, createdTime, onEditTodo , isCompleted}) {
+  const [isChecked, setIsChecked] = useState(isCompleted);
+  const [isEditing, setIsEditing] = useState(false);
+  const [updatedTodoTitle, setUpdatedTodoTitle] = useState("");
```
Comment (AnnaPestova1) [inline src/Components/TodoListItem/TodoListItem.jsx]: Can you pre-populate input with the current data, what do you think? 

https://github.com/user-attachments/assets/c6bea5c5-593a-4642-ab40-49e4418ff685

File: src/Components/TodoListItem/TodoListItem.jsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+          ) : (
+            <span
+              style={{ textDecoration: isCompleted ? "line-through" : "none" }}
+            >
+              {title}
+            </span>
+          )}
+
+          <span className={styles.date}>
+            {new Date(createdTime).toLocaleString("en-US", {
```
Comment (AnnaPestova1) [inline src/Components/TodoListItem/TodoListItem.jsx:61]: great handle Data

**Review by AnnaPestova1 — APPROVED** [review]
Hi Nihal! Your code looks awesome. You need to handle checkbox and it is better to show the default value in inputs during editing todo.
Approved

File: src/Components/TodoListItem/TodoListItem.jsx (hunk 1 above)
Comment (nihalerdal) [reply src/Components/TodoListItem/TodoListItem.jsx]: fixed!

**Pull request merged**